dot/parachain/backing/statement_table.go

```diff
@@ -67,10 +121,35 @@ func (statementTable) importStatement( //nolint:unused
 	return nil, nil
 }
 
-func (statementTable) attestedCandidate(candidateHash parachaintypes.CandidateHash, ctx *TableContext, //nolint:unused
-) (*AttestedCandidate, error) {
-	// TODO: Implement this method
-	return nil, nil
+// attestedCandidate retrieves the attested candidate for the given candidate hash.
+// returns attested candidate  if the candidate exists and is includable.
+func (table statementTable) attestedCandidate(
+	candidateHash parachaintypes.CandidateHash, tableContext *TableContext, minimumBackingVotes uint32,
```

Comment:
I'm curious why `minimumBackingVotes` is a `uint32` when it seems that when it's used it is cast to a `uint`?

Reply:
in polkadot-sdk they have used u32 type for minimumBackingVotes in [this](https://github.com/paritytech/polkadot-sdk/blob/7ca0d65f19497ac1c3c7ad6315f1a0acb2ca32f8/polkadot/node/core/backing/src/lib.rs#L206-L207-226) struct